hadoop-hdds/interface-admin/src/main/proto/ScmAdminProtocol.proto

```diff
@@ -628,19 +628,20 @@ message ContainerBalancerStatusInfo {
 message ContainerBalancerTaskIterationStatusInfo {
```

Comment:
Note that all other messages in this file have the suffix "Proto". This really improves code readability because the reader can distinguish between the java class `ContainerBalancerTaskIterationStatusInfo` and the java class that's generated by protoc for the proto message.